core/src/main/scala/kafka/server/KafkaConfig.scala

```diff
@@ -894,7 +894,7 @@ object KafkaConfig {
     "record batch size accepted by the broker is defined via <code>message.max.bytes</code> (broker config) or " +
     "<code>max.message.bytes</code> (topic config)."
   val NumReplicaFetchersDoc = "Number of fetcher threads used to replicate messages from a source broker. " +
```

Comment:
I think the one thing this description should emphasize is that the value represents the number of fetchers for each broker that we are fetching from. This is implied by the phrase "from a source broker," but I can we can make it more explicit. How about this?

> Number of fetcher threads used to replicate records from each source broker. The total number of fetchers on each broker is bound by `num.replica.fetchers` multiplied by the number of brokers in the cluster. Increasing this value can increase the degree of I/O parallelism in the follower and leader broker at the cost of higher CPU/memory utilization.

Reply:
i have updated the PR https://github.com/apache/kafka/pull/12153/commits/0957f0881c7638c533ff2f0ffbc083f58448bc79